content/stable-rc/index.html

```diff
@@ -10,8 +10,6 @@
 <p>
 LTS Release candidate is the build intended for the upcoming LTS release of Jenkins. <a href="https://wiki.jenkins-ci.org/display/JENKINS/Release+Process">See Wiki for more about what it means.</a>
 </p>
-<p>Upcoming version: <b><?php include "/srv/releases/jenkins/version-stable-rc.txt"?></b></p>
```

Comment:
Yes, will implement that once I figure out what to do: http://lists.jenkins-ci.org/pipermail/jenkins-infra/2016-February/000595.html. Meanwhile this is outright useless.